### PR TITLE
Adds two new traits: Air Sickness and Lactose Intolerance

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Adds two new traits: Air Sickness and Lactose Intolerance.

Air Sickness will induce nausea when you're on a ship and it either moves or some other heavy impact happens.

Lactose Intolerance will cause milk and cheese (dairy) products to give you an increasingly bad set of effects. Note: There's a lot of diary products in places you might not even notice, including MREs...

Thess traits don't cost points because they're very understated and inoffensive 

Adds a new variable to reagents, current_cycle, that tracks how many times the reagent has been processed.

Fixed a runtime on the bad leg trait.

Added milk or cheese to the dairy products that did not contain it.

I think something's up with this PR, I kept getting endless loop errors because of the lactose property for some reason so please TM before merging.

# Explain why it's good for the game

> Adds two new traits: Air Sickness and Lactose Intolerance.

> Air Sickness will induce nausea when you're on a ship and it either moves or some other heavy impact happens.

People get sick sometimes when in air. POs cannot get this trait and neither can DCCs, but other marines can because, well, once the nausea passes they're still combat effective. This will add a little noticeable flavor to your character.

> Lactose Intolerance will cause milk and cheese (dairy) products to give you an increasingly bad set of effects. Note: There's a lot of diary products in places you might not even notice, including MREs...

None of these effects are deadly. Vomit, eye blurriness are as bad as they get. Realizing that you accidentally ate a honey flan or cheese spread from an MRE and then getting woozy with people noticing could be funny, same as some malicious person serving you 'lactose free' food.

> Adds a new variable to reagents, current_cycle, that tracks how many times the reagent has been processed.

Needed for over-time effects.

> Fixed a runtime on the bad leg trait.

It was annoying me!

>  Added milk or cheese to the dairy products that did not contain it.

Necessary consistency.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:

add: Adds two new traits: Air Sickness and Lactose Intolerance.
add: Air Sickness will induce nausea when you're on a ship and it either moves or some other heavy impact happens.
add: Lactose Intolerance will cause milk and cheese (dairy) products to give you an increasingly bad set of effects. Note: There's a lot of diary products in places you might not even notice, including MREs...
code: Adds a new variable to reagents, current_cycle, that tracks how many times the reagent has been processed.
fix: Fixed a runtime on the bad leg trait.
code: Added milk or cheese to the dairy products that did not contain it.
add: Reduced vomit stun to 1.5 seconds.
/:cl:

### CODE REVIEW

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -362,7 +362,7 @@
 
 /obj/item/reagent_container/food/snacks/chocolatebar/Initialize()
 	. = ..()
-	reagents.add_reagent("nutriment", 2)
+	reagents.add_reagent("milk", 2)
 	reagents.add_reagent("sugar", 2)
```
Comment (carlarctg) [inline code/game/objects/items/reagent_containers/food/snacks.dm:366]:
```suggestion
	reagents.add_reagent("nutriment", 2)
```

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Ahem, 1 point per trait.

**Review by morrowwolf — COMMENTED** [review]
Waiting for #2756 to be merged and then will get to this as it causes ship shakes. Remind me after.

**Review by harryob — CHANGES_REQUESTED** [review]
the thing above

File: code/modules/shuttles/shuttle.dm [hunk 2]
```diff
@@ -211,8 +211,12 @@
 					to_chat(M, SPAN_WARNING("Sudden acceleration presses you into [M.buckled]!"))
 					shake_camera(M, 3, 1)
 				else if (!M.buckled)
-					to_chat(M, SPAN_WARNING("The floor lurches beneath you!"))
```
Comment (harryob) [inline code/modules/shuttles/shuttle.dm:214]: ?

**Review by harryob — APPROVED** [review]
looks fine but

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**